components/src/dynamo/frontend/flexprice/proxy.py

```
@@ -0,0 +1,385 @@
+                        start=start,
+                    )
+                else:
+                    return await self._handle_buffered(
+                        backend_resp,
+                        resp_headers,
+                        is_metered=is_metered,
+                        org_id=org_id,
+                        model_name=model_name,
+                        start=start,
+                    )
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)